### ISSUE TITLE
What is Overview for?

### ISSUE THREAD

**Issue by amazansky** [PlanHubMe/PlanHub#234]
What is the Overview tab for?
It just has the date.

This is probably low priority, but just wondering?
You could also add the date to MyDay and have that be the default view?

**Comment by willbarkoff** [comment]
Hi @RobotXLabs,

The overview feature has not been completed yet. It will eventually contain information about your upcoming assignments, and possibly your schedule. PlanHub is also in a public beta phase, and is in no means a finished project. If you have any ideas about what content can go on the overview page, you can tell us by replying to this issue or clicking the lightbulb button in the PlanHub app.

As for MyDay, that could be possible once MyDay is finished.

Thanks for your feedback!

**Comment by amazansky** [comment]
Also @nywillb this is me (c22am)

**Comment by willbarkoff** [comment]
@RobotXLabs I know, I looked at your profile and saw RobotXLabs/back-to-drive.
